src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[];
+  filter: number;
+  setFilter: (value: number) => void;
+  handleClearCompleted: () => void;
+};
+export const Footer: React.FC<Props> = ({
+  todos,
+  filter,
+  setFilter,
+  handleClearCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {[...todos].filter(todo => !todo.completed).length} items left
```

Comment:
Move this logic from jsx to the helper variable and use it here
```suggestion
        {todos.filter(todo => !todo.completed).length} items left
```